pkg/v1/tkg/test/data/testdata/customClusterBootstrapAntrea.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: cni.tanzu.vmware.com/v1alpha1
+kind: AntreaConfig
+metadata:
+  name: cc20-antrea-package
```

Comment:
nit: why the name has 20 suffix?:)

Reply:
I can use another name => cc-cb. Let me know if thats fine or Please suggest a name you like.

Reply:
that sounds good!

Reply:
Changed it to cc-cb